app/jobs/reports/fraud_metrics_report.rb

```diff
@@ -0,0 +1,94 @@
+require 'csv'
+
+module Reports
+  class FraudMetricsReport < BaseReport
+    REPORT_NAME = 'fraud-metrics-report'.freeze
+
+    attr_reader :report_date
+
+    def initialize(report_date = nil, *args, **rest)
+      @report_date = report_date
+      super(*args, **rest)
+    end
+
+    def perform(date = Time.zone.yesterday.end_of_day)
+      @report_date = date
+
+      email_addresses = emails.select(&:present?)
+      if email_addresses.empty?
+        Rails.logger.warn 'No email addresses received - Fraud Metrics Report NOT SENT'
+        return false
+      end
+
+      reports.each do |report|
+        upload_to_s3(report.table, report_name: report.filename)
+      end
+
+      ReportMailer.tables_report(
+        email: email_addresses,
+        subject: "Fraud Metrics Report - #{date.to_date}",
+        reports: reports,
+        message: preamble,
+        attachment_format: :xlsx,
+      ).deliver_now
+    end
+
+    # Explanatory text to go before the report in the email
+    # @return [String]
+    def preamble(env: Identity::Hostdata.env || 'local')
+      ERB.new(<<~ERB).result(binding).html_safe # rubocop:disable Rails/OutputSafety
+        <% if env != 'prod' %>
+          <div class="usa-alert usa-alert--info usa-alert--email">
+            <div class="usa-alert__body">
+              <%#
+                NOTE: our AlertComponent doesn't support heading content like this uses,
+                so for a one-off outside the Rails pipeline it was easier to inline the HTML here.
+              %>
+              <h2 class="usa-alert__heading">
+                Non-Production Report
+              </h2>
+              <p class="usa-alert__text">
+                This was generated in the <strong><%= env %></strong> environment.
+              </p>
+            </div>
+          </div>
+        <% end %>
+        <p>
+        </p>
+      ERB
+    end
+
+    def reports
+      @reports ||= []
+    end
+
+    def emails
+      emails = [*IdentityConfig.store.test_fraud_reports_emails]
+      if report_date.next_day.day == 1
+        emails += IdentityConfig.store.team_fraud_reports_emails
+      end
```

Comment:
is this a weekly or a monthly report?

Reply:
Just to wrap this up - I think we can leave monthly off of the file name, in case we want to change how often we deliver these in the future.